Solutions/Illusive Active Defense/Analytic Rules/Illusive_Detection_Query.yaml

```diff
@@ -0,0 +1,46 @@
+id: 1a7dbcf6-21a2-4255-84b2-c8dbbdca4630
+name: Illusive Incidents Analytic Rule
+description: |
+  'Create a Sentinel incident upon a new Illusive alert (incident) and associate all related Illusive events to the relevant Sentinel alert. This is done by filtering and processing Illusive Syslog messages.'
+severity: Medium
+requiredDataConnectors: 
+  - connectorId: Illusive
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
```

Comment:
can you add any `Tactics` and `relevantTechniques` based on available alerts ?   
You can put multiple values as a list.

Reply:
we are currently not using Tactics and relevantTechniques as per the requirement, removed these fields from the rule.
Thanks